CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
OMG - I hate this particular code of conduct so much - it's so full of vaguely threatening but finally toothless legaleze.   Just humbly asking - consider another one that isn't like that?

Reply:
Here's one for our set of brain imaging projects.  I think it's OK - and that it avoids most of the arsey stuff that so often pollutes codes of conducts : https://github.com/nipy/nibabel/blob/master/.github/CODE_OF_CONDUCT.md

Reply:
I've never put much thought into codes of conduct - but I have now! I see your point so I've made a Contributor Covenant-Nipy hybrid.